libkpod/container_server.go

```diff
@@ -579,8 +579,9 @@ func (c *ContainerServer) Shutdown() error {
 }
 
 type containerServerState struct {
-	containers oci.ContainerStorer
-	sandboxes  map[string]*sandbox.Sandbox
+	containers      oci.ContainerStorer
+	infraContainers oci.ContainerStorer
```

Comment:
Everything looks good except cri-o is having a panic in tests because this is uninitialized, you need to initialize it here https://github.com/mrunalp/cri-o/blob/948f16cd046e3071c98a29fa5c477e1aa95f0c2c/libkpod/container_server.go#L147

Reply:
Fixing.